sunpy/visualization/colormaps/cm.py

```diff
@@ -142,6 +155,10 @@
     'euvi195': euvi195,
     'euvi284': euvi284,
     'euvi304': euvi304,
+    'solar orbiterfsi174': solofsi174,
+    'solar orbiterfsi304': solofsi304,
+    'solar orbiterhri_euv174': solohri_euv174,
+    'solar orbiterhri_lya1216': solohri_lya1216,
```

Comment:
suggestion to change key to  `solo_fsi/hri_wavelength`

Reply:
The keys have to be this, as they're automatically constructed from the map metadata (in `GenericMap._get_cmap_name()`). I could change the colormap variable names though to add an extra underscore?

Reply:
ah I see, I thought it was weird with the whitespace, but that makes sense! never mind :)